src/model/FaceliftModel.h

```diff
@@ -39,7 +39,6 @@
 #include <qqml.h>
 #include <QPointer>
 
-
```

Comment:
No substantial change, pleas remove

Reply:
I squashed commits but for some reason CI queued a job but the status has been pending for some time. Should I do something about that?